build/configs/rtl8721csm/cxxtest/Make.defs

```diff
@@ -0,0 +1,229 @@
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
2021

Reply:
How about this?? Is this also need to change 2021?? 

```
#   Copyright (C) 2020 Gregory Nutt. All rights reserved.
#   Author: Gregory Nutt <gnutt@nuttx.org>
```

Reply:
No. You should not modify Nuttx license itself.